Makefile

```diff
@@ -73,7 +73,10 @@ watch:
 cross:
 	CGO_ENABLED=0 GOOS=linux go build -o build/bin/linux/$(BINARY_NAME) $(GOBUILD_VERSION_ARGS) -a -installsuffix cgo  github.com/jtblin/$(BINARY_NAME)
 
-docker: cross
+cross-docker:
+	docker run --rm -v "$(GOPATH)":/opt/gopath -w /opt/gopath/src/github.com/jtblin/$(BINARY_NAME) -e GOPATH=/opt/gopath golang:latest go build -o build/bin/linux/$(BINARY_NAME) $(GOBUILD_VERSION_ARGS) -a -installsuffix cgo github.com/jtblin/$(BINARY_NAME)
```

Comment:
I wanted to use `-race` to build it but got an error:
```
go build: -race requires cgo; enable cgo by setting CGO_ENABLED=1
```
After enabling it:
```
CGO_ENABLED=1 GOOS=linux go build -race -o build/bin/linux/kube2iam -ldflags "-s -X ""github.com/jtblin"/kube2iam"/version.Version=$(git describe --abbrev=0 --tags) -X ""github.com/jtblin"/kube2iam"/version.GitCommit=$(git rev-parse --short HEAD) -X ""github.com/jtblin"/kube2iam"/version.BuildDate=$(date +%Y-%m-%d-%H:%M)" -a -installsuffix cgo  github.com/jtblin/kube2iam
# runtime/cgo
ld: unknown option: --build-id=none
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [cross] Error 2
```

With this change the build happens in a docker container `golang:latest` i.e. in Linux environment. No need to have crazy cross-compilation setup locally. I think it is more reliable to use linux toolchain than using OS X compiler toolchain to build for linux.